libs/components/lists/src/lib/modules/shared/filter-adapter/filter-adapter-filter-value.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Represents a value for a filter item.
+ */
+export interface SkyFilterAdapterFilterValue {
+  /**
+   * The real value for the filter.
+   */
+  value: unknown;
+  /**
+   * A human-readable string for use with values that can't be displayed to the user.
+   */
+  displayValue?: string;
```

Comment:
Can we make this property required? Is there a use-case for a filter value to only have a raw value and not a display value?

Reply:
If the value for a filter is a string that can be displayed to the user, then displayValue is superfluous. This was a convenience factor for the consumer